dist/pagination.js

```diff
@@ -356,7 +373,7 @@
         postData[alias.pageSize ? alias.pageSize : 'pageSize'] = pageSize;
         postData[alias.pageNumber ? alias.pageNumber : 'pageNumber'] = pageNumber;
 
-        var ajaxParams = $.isFunction(attributes.ajax) ? attributes.ajax() : {};
+        var ajaxParams = $.isFunction(attributes.ajax) ? attributes.ajax() : attributes.ajax;
```

Comment:
It will not work if attributes don't have ajax option.
I don't offer the best solution, but you can check it:
```javascript
var ajaxParams = $.isFunction(attributes.ajax) ? attributes.ajax() : (attributes.ajax || {});
```